src/core/friendly_errors/fes_core.js

```diff
@@ -303,33 +316,186 @@ if (typeof IS_MINIFIED !== 'undefined') {
       }
     });
 
-    if (min > EDIT_DIST_THRESHOLD) return;
+    if (min > Math.min(EDIT_DIST_THRESHOLD, errSym.length)) return false;
 
     let symbol = misusedAtTopLevelCode[minIndex];
 
     // Show a message only if the caught symbol and the matched property name
     // differ in their name ( either letter difference or difference of case )
     if (errSym !== symbol.name) {
       const parsed = p5._getErrorStackParser().parse(error);
-      const location =
-        parsed[0] && parsed[0].fileName
-          ? `${parsed[0].fileName}:${parsed[0].lineNumber}:${
-              parsed[0].columnNumber
-            }`
-          : null;
+      let locationObj;
+      if (
+        parsed &&
+        parsed[0] &&
+        parsed[0].fileName &&
+        parsed[0].lineNumber &&
+        parsed[0].columnNumber
+      ) {
+        locationObj = {
+          location: `${parsed[0].fileName}:${parsed[0].lineNumber}:${
+            parsed[0].columnNumber
+          }`,
+          file: parsed[0].fileName,
+          line: parsed[0].lineNumber
+        };
+      }
       const msg = translator('fes.misspelling', {
         name: errSym,
         actualName: symbol.name,
         type: symbol.type,
-        location: location ? translator('fes.location', { location }) : ''
+        location: locationObj ? translator('fes.location', locationObj) : ''
       });
 
-      if (log) {
-        log(msg);
+      p5._friendlyError(msg, symbol.name);
+      return true;
+    }
+    return false;
+  };
+
+  const processStack = (error, stacktrace) => {
+    // Responsible for removing internal library calls from the stacktrace
+    // and also for detectiong if the error happened inside the library
+
+    // cannot process a stacktrace that doesn't exist
+    if (!stacktrace) return [false, null];
+
+    stacktrace.forEach(frame => {
+      frame.functionName = frame.functionName || '';
+    });
+
+    // isInternal - Did this error happen inside the library
+    let isInternal = false;
+    let p5FileName, friendlyStack, currentEntryPoint;
+    for (let i = stacktrace.length - 1; i >= 0; i--) {
+      let splitted = stacktrace[i].functionName.split('.');
+      if (entryPoints.includes(splitted[splitted.length - 1])) {
+        // remove everything below an entry point function (setup, draw, etc).
+        // (it's usually the internal initialization calls)
+        friendlyStack = stacktrace.slice(0, i + 1);
+        currentEntryPoint = splitted[splitted.length - 1];
+        for (let j = 0; j < i; j++) {
+          // Due to the current build process, all p5 functions have
+          // _main.default in their names in the final build. This is the
+          // easiest way to check if a function is inside the p5 library
+          if (stacktrace[j].functionName.search('_main.default') !== -1) {
```

Comment:
Yes. I will also add this to the FES docs when the documentation is updated. I can't seem to find docs about how the entire build process works. This could be added there too as a note if it exists.